src/main/java/io/iron/ironworker/client/Client.java

```diff
@@ -213,9 +222,17 @@ public ScheduleEntity createSchedule(String codeName, Map<String, Object> params
         if (params == null) {
             params = new HashMap<String, Object>();
         }
+        String payload = gson.toJson(params);
+        if(options!=null){
+        	String encryptionKeyFile = (String) options.get("encryptionKeyFile");
```

Comment:
use constant in TaskOptionsObject.ENCRYPTION_KEY_FILE instead of `"encryptionKeyFile"`